### DISCUSSION TITLE
Send Later not working with Proton Mail?

### DISCUSSION THREAD

**Discussion by NathanVeenstra** [Extended-Thunder/send-later#562] [Q&A]
I am having issues with Send Later since several months and I'm not 100% sure whether it coincided with [the release of scheduled emails](https://proton.me/blog/schedule-send) in Proton Mail. I just cannot seem to fix it. 

I have tried to uninstall and re-install Send Later, changed settings, et cetera, but nothing seems to work. 

I'm on:
- Windows 10
- Thunderbird 102.12.0 (64 bits)
- Send Later 9.2.9

Since I'm not a techie, I am not sure whether I should provide more or other information, nor do I have any clue what else I can do. It's really frustrating as I do want to use Send Later and I can't. 

Can anyone help please?

**Comment by jikamens** [comment]
Can you explain in more detail exactly what the issues you're having are? What exactly in the process is failing? For example:

* Are you able to schedule a message with Send Later?
* When you've done so, does the scheduled send time show up in the "Send Later" column in the message list?
* Does the message send at the scheduled time?
* Are you seeing any error messages, and if so, what exactly do they say?

Your description above is too vague to be actionable for us. We need more details. Thanks!

**Comment by jikamens** [comment]
Unfortunately this is difficult for us to attempt to reproduce because Proton Mail does not support IMAP and SMTP directly. It only supports them through Proton Mail Bridge, which is only supported with paid Proton Mail accounts, i.e., I can't test using Send Later with Proton Mail unless I pay Proton Mail money, which I otherwise have no need to do.

**Reply by NathanVeenstra** [reply]
If there's anything you want me to do or test, just let me know. As long as you're aware that I'm only a bit more tech savvy than the average person. Plus English is not my native language, which is what makes tech stuff a bit harder for me.

**Comment by NathanVeenstra** [comment]
Thanks for your reply and helping me to draw the situation more clearly by sending me these questions. I wasn't aware what details were necessary, so here we go:

- Yes I can schedule a message, but it does not send when it's supposed to be sent.
- I do not use a "Send Later" column, so I can't say whether send time shows up.
- The message does not send at the scheduled time, it just stays in the concepts folder.
- No error message. 

I hope this helps. If you need more details, I will have to try and schedule a message again. I will not be able to do that this week as we're off for a short holiday.

**Comment by jikamens** [comment]
OK, let's try to narrow down what's going wrong. When you get a chance, try scheduling a message, then find the scheduled message in your drafts folder, click on it in the message list, and type ctrl-u to open the source code for the message. Check the header of the message in the source window to see which (if any) "X-Send-Later-*" header fields are in it. You should see headers that look like this:

```
X-Send-Later-Uuid: [elided]
X-Send-Later-At: Wed, 26 Jul 2023 17:09:00 -0400
X-Send-Later-Recur: none
```

Let us know what you see.

**Comment by jikamens** [comment]
I can confirm this is a Proton Mail Bridge issue. Their IMAP server isn't returning the Send Later headers it should be. This has been documented in the user guide and I'm tracking it in an [open issue](https://github.com/Extended-Thunder/send-later/issues/586), though the fix needs to be made on the Proton side. In the meantime the only workaround is to store your Drafts for your Proton Mail account in a local folder rather than on their server. Or I suppose if you have another IMAP account you could store the drafts there; it's just not going to work as long as your drafts are stored in your Proton account, unfortunately, until they fix it.
